kili/mutations/asset/helpers.py

```diff
@@ -314,3 +354,56 @@ def generate_json_metadata_array(as_frames, fps, nb_files, input_type):
             }
         ] * nb_files
     return json_metadata_array
+
+
+def request_signed_urls(auth, project_id: str, size: int):
+    """
+    Get upload signed URLs
+    Args:
+        auth: Kili Auth
+        project_id: the project Id
+        size: the amount of upload signed URL to query
+    """
+    payload = {
+        "projectID": project_id,
+        "size": size,
+    }
+    urls_response = auth.client.execute(GQL_CREATE_UPLOAD_BUCKET_SIGNED_URLS, payload)
+    return urls_response["data"]["urls"]
+
+
+def upload_data_via_rest(signed_urls, data_array: List[str], content_type_array: List[str]):
```

Comment:
add type to signed_urls

Reply:
note: I think that, moving forward, we should handle list of typed dicts instead of several arrays, until we call the resolvers where data can be split. I know this will require some back and forth to maintain the V2 API, but the code will be more readable and debuggable.